documentation/Extensions.md

```diff
@@ -140,7 +141,7 @@ Extensions and legacy features deliberately not supported
 * Defining an explicit interface for a subprogram within itself (PGI only)
 * USE association of a procedure interface within that same procedure's definition
 * NULL() as a structure constructor expression for an ALLOCATABLE component (PGI).
-* Conversion of LOGICAL to INTEGER.
+* Conversion of LOGICAL to INTEGER in expressions.
```

Comment:
The other compilers that support this (PGI/Intel) also allow assignment between LOGICAL and other numeric types. If we're not supporting it we should mention that here.